update_deps.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# This script updates all checked-in Manifest.toml files to the
+# latest versions of all dependencies.
+# It requires that `juliaup` is installed.
+
+# Update checked-in Manifests for Julia v1.10
+juliaup add 1.10.5
+juliaup default 1.10.5
+# julia --project=experiments/ClimaCore/ -e 'using Pkg; Pkg.update()'
+# julia --project=experiments/ClimaCore/sea_breeze/ -e 'using Pkg; Pkg.update()'
+# julia --project=experiments/ClimaEarth/ -e 'using Pkg; Pkg.update()'
+julia --project=perf/ -e 'using Pkg; Pkg.update()'
+
+# Update checked-in Manifests for Julia v1.11
+juliaup add 1.11.1
+juliaup default 1.11.1
+# julia --project=experiments/ClimaCore/ -e 'using Pkg; Pkg.update()'
+# julia --project=experiments/ClimaCore/sea_breeze/ -e 'using Pkg; Pkg.update()'
+# julia --project=experiments/ClimaEarth/ -e 'using Pkg; Pkg.update()'
+julia --project=perf/ -e 'using Pkg; Pkg.update()'
```

Comment:
I use juliaup differently, so I personally wouldn't use this script. Also, this script would precompile all the packages and take a while. You can set `JULIA_PKG_PRECOMPILE_AUTO=0` to avoid those precompilation

Previously, I was using Charlie's PkgDevTools, but this doesn't work with Julia 1.11 (https://github.com/CliMA/PkgDevTools.jl/issues/11).

Reply:
Thanks, I'll set `JULIA_PKG_PRECOMPILE_AUTO=0`.

Do you have any suggestions for how to change the juliaup usage? Even if some of the developers have different ways of using it, I think it's helpful to have some script that automates the updates.